L1Trigger/Phase2L1ParticleFlow/python/DeregionizerProducer_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
-DeregionizerProducer = cms.EDProducer("DeregionizerProducer",
-                           RegionalPuppiCands  = cms.InputTag("l1ctLayer1","PuppiRegional"),
+l1tDeregionizerProducer = cms.EDProducer("DeregionizerProducer",
```

Comment:
Duplicate of
cfipython/L1Trigger/Phase2L1ParticleFlow/DeregionizerProducer_cfi.py

with the same search path

Reply:
Done